mm2src/coins/z_coin.rs

```diff
@@ -184,6 +192,47 @@ pub struct ZOutput {
     pub memo: Option<MemoBytes>,
 }
 
+struct ZCoinSqlTxHistoryItem {
+    tx_hash: Vec<u8>,
+    internal_id: i64,
```

Comment:
I see that you used `i64` here since it's the native integer type for SQLite. Do you think it's better to change this to `u64` after updating rusqlite or just keep it as is? By my understanding it doesn't make a difference and I might not need to update rusqlite and use `i64` also in the similar cases I faced, but just checking 🙂

Reply:
Yes, we can change it to u64, won't make much difference, though 🙂